Libs/Widgets/ctkFileDialog.cpp

```diff
@@ -67,6 +67,11 @@ void ctkFileDialogPrivate::init()
 {
   Q_Q(ctkFileDialog);
 
+  if (!(q->options() & QFileDialog::DontUseNativeDialog))
```

Comment:
What happens if donusenativedialog flag is set after init()? If you don't want to allow changing native/non-native flag after initialization (which may be a good idea, because it is probably rarely needed and it would take some effort to implement correctly) then it would be safer to store a the value of the DontUseNativeDialog flag in a member variable and use that (instead of checking options(), which may change).

Reply:
I've pushed an additional commit with changes to a member variable. See if this matches what you were thinking.